ballerina/annotations.bal

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# Description.
+# This annotation is used to map an entity/field name to a database table/column name.
+# + name - name of the table/column in the database
+public type MapConfig record {|
+    string name;
+|};
+
+public annotation MapConfig Mapping on type, record field;
```

Comment:
Shall we add comments like below,
https://github.com/ballerina-platform/module-ballerina-sql/blob/master/ballerina/annotation.bal#L17-L25